--- ./node_modules/app-builder-lib/out/macPackager.js	1985-10-26 09:15:00.000000000 +0100
+++ ./node_modules/app-builder-lib/new/macPackager.js	2019-10-14 12:50:02.000000000 +0200
@@ -315,7 +315,8 @@
         keychain: keychainName || undefined,
         binaries: (isMas && masOptions != null ? masOptions.binaries : macOptions.binaries) || undefined,
         requirements: isMas || macOptions.requirements == null ? undefined : yield _this3.getResource(macOptions.requirements),
-        "gatekeeper-assess": _codeSign().appleCertificatePrefixes.find(it => identity.name.startsWith(it)) != null
+        "gatekeeper-assess": _codeSign().appleCertificatePrefixes.find(it => identity.name.startsWith(it)) != null,
+        "hardened-runtime": macOptions.hardenedRuntime
       };
       yield _this3.adjustSignOptions(signOptions, masOptions);
 
@@ -471,6 +472,6 @@
   }
 
   return isMas ? "3rd Party Mac Developer Application" : "Developer ID Application";
-} 
+}
 // __ts-babel@6.0.4
-//# sourceMappingURL=macPackager.js.map
\ No newline at end of file
+//# sourceMappingURL=macPackager.js.map
